### ISSUE TITLE
pulp-cli breaks after pulp-cli-maven is installed

### ISSUE THREAD

**Issue by av-gal** [pulp/pulp-cli-maven#53]
## Summary
Pulp-cli breaks after the pulp-cli-maven plugin is installed

## Steps to reproduce

1. `pipx install pulp-cli`
2. `pipx inject pulp-cli pulp-cli-maven`
3. `pulp status`

## Expected behavior
`pulp status` should print the pulp server's status.

## Stacktrace/Error log
```Traceback (most recent call last):
  File "/home/alex/.local/bin/pulp", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/alex/.local/pipx/venvs/pulp-cli/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/alex/.local/pipx/venvs/pulp-cli/lib/python3.11/site-packages/click/core.py", line 1077, in main
    with self.make_context(prog_name, args, **extra) as ctx:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/alex/.local/pipx/venvs/pulp-cli/lib/python3.11/site-packages/click/core.py", line 943, in make_context
    self.parse_args(ctx, args)
  File "/home/alex/.local/pipx/venvs/pulp-cli/lib/python3.11/site-packages/click/core.py", line 1644, in parse_args
    rest = super().parse_args(ctx, args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/alex/.local/pipx/venvs/pulp-cli/lib/python3.11/site-packages/click/core.py", line 1408, in parse_args
    value, args = param.handle_parse_result(ctx, opts, args)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/alex/.local/pipx/venvs/pulp-cli/lib/python3.11/site-packages/click/core.py", line 2400, in handle_parse_result
    value = self.process_value(ctx, value)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/alex/.local/pipx/venvs/pulp-cli/lib/python3.11/site-packages/click/core.py", line 2362, in process_value
    value = self.callback(ctx, self, value)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/alex/.local/pipx/venvs/pulp-cli/lib/python3.11/site-packages/pulp_cli/__init__.py", line 75, in _config_profile_callback
    _load_config(ctx)
  File "/home/alex/.local/pipx/venvs/pulp-cli/lib/python3.11/site-packages/pulp_cli/__init__.py", line 116, in _load_config
    ctx.meta[PLUGIN_KEY] = load_plugins(enabled_plugins)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/alex/.local/pipx/venvs/pulp-cli/lib/python3.11/site-packages/pulp_cli/__init__.py", line 39, in load_plugins
    discovered_plugins: t.Dict[str, ModuleType] = {
                                                  ^
  File "/home/alex/.local/pipx/venvs/pulp-cli/lib/python3.11/site-packages/pulp_cli/__init__.py", line 40, in <dictcomp>
    entry_point.name: entry_point.load()
                      ^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/metadata/__init__.py", line 202, in load
    module = import_module(match.group('module'))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1206, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1178, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1149, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/alex/.local/pipx/venvs/pulp-cli/lib/python3.11/site-packages/pulpcore/cli/maven/__init__.py", line 7, in <module>
    from pulpcore.cli.maven.distribution import distribution
  File "/home/alex/.local/pipx/venvs/pulp-cli/lib/python3.11/site-packages/pulpcore/cli/maven/distribution.py", line 3, in <module>
    from pulp_glue.maven.context import (
  File "/home/alex/.local/pipx/venvs/pulp-cli/lib/python3.11/site-packages/pulp_glue/maven/context.py", line 4, in <module>
    from pulp_glue.common.context import (
ImportError: cannot import name 'registered_repository_contexts' from 'pulp_glue.common.context' (/home/alex/.local/pipx/venvs/pulp-cli/lib/python3.11/site-packages/pulp_glue/common/context.py)
```
## Pulp and pulp-cli version info
```
alex@debian:~$ pulp --version
Pulp3 Command Line Interface, Version 0.25.2
Plugin Versions:
  common: 0.25.2
```
```
alex@debian:~$ pulp status
{
  "versions": [
    {
      "component": "core",
      "version": "3.54.0.dev",
      "package": "pulpcore",
      "module": "pulpcore.app",
      "domain_compatible": true
    },
    {
      "component": "ansible",
      "version": "0.22.0.dev",
      "package": "pulp-ansible",
      "module": "pulp_ansible.app",
      "domain_compatible": false
    },
    {
      "component": "container",
      "version": "2.21.0.dev",
      "package": "pulp-container",
      "module": "pulp_container.app",
      "domain_compatible": false
    },
    {
      "component": "deb",
      "version": "3.3.0.dev",
      "package": "pulp_deb",
      "module": "pulp_deb.app",
      "domain_compatible": false
    },
    {
      "component": "gem",
      "version": "0.6.0.dev",
      "package": "pulp-gem",
      "module": "pulp_gem.app",
      "domain_compatible": true
    },
    {
      "component": "maven",
      "version": "0.9.0.dev",
      "package": "pulp-maven",
      "module": "pulp_maven.app",
      "domain_compatible": false
    },
    {
      "component": "npm",
      "version": "0.1.0a5.dev",
      "package": "pulp-npm",
      "module": "pulp_npm.app",
      "domain_compatible": false
    },
    {
      "component": "ostree",
      "version": "2.4.0.dev",
      "package": "pulp-ostree",
      "module": "pulp_ostree.app",
      "domain_compatible": true
    },
    {
      "component": "python",
      "version": "3.12.0.dev",
      "package": "pulp-python",
      "module": "pulp_python.app",
      "domain_compatible": false
    },
    {
      "component": "rpm",
      "version": "3.26.0.dev",
      "package": "pulp-rpm",
      "module": "pulp_rpm.app",
      "domain_compatible": true
    },
    {
      "component": "certguard",
      "version": "3.54.0.dev",
      "package": "pulpcore",
      "module": "pulp_certguard.app",
      "domain_compatible": true
    },
    {
      "component": "file",
      "version": "3.54.0.dev",
      "package": "pulpcore",
      "module": "pulp_file.app",
      "domain_compatible": true
    }
  ],
  "online_workers": [
    {
      "pulp_href": "/pulp/api/v3/workers/018fa185-0a8d-7268-8f91-b5f3a6697480/",
      "pulp_created": "2024-05-22T18:16:41.874326Z",
      "pulp_last_updated": "2024-05-22T18:16:41.874360Z",
      "name": "1@example-pulp-worker-7d4875fff4-n7kt5",
      "last_heartbeat": "2024-05-22T20:24:20.296172Z",
      "versions": {
        "deb": "3.3.0.dev",
        "gem": "0.6.0.dev",
        "npm": "0.1.0a5.dev",
        "rpm": "3.26.0.dev",
        "core": "3.54.0.dev",
        "file": "3.54.0.dev",
        "maven": "0.9.0.dev",
        "ostree": "2.4.0.dev",
        "python": "3.12.0.dev",
        "ansible": "0.22.0.dev",
        "certguard": "3.54.0.dev",
        "container": "2.21.0.dev"
      },
      "current_task": null
    }
  ],
  "online_api_apps": [
    {
      "name": "21@example-pulp-api-7566944f48-t22q5",
      "last_heartbeat": "2024-05-22T20:24:29.747514Z",
      "versions": {
        "deb": "3.3.0.dev",
        "gem": "0.6.0.dev",
        "npm": "0.1.0a5.dev",
        "rpm": "3.26.0.dev",
        "core": "3.54.0.dev",
        "file": "3.54.0.dev",
        "maven": "0.9.0.dev",
        "ostree": "2.4.0.dev",
        "python": "3.12.0.dev",
        "ansible": "0.22.0.dev",
        "certguard": "3.54.0.dev",
        "container": "2.21.0.dev"
      }
    },
    {
      "name": "20@example-pulp-api-7566944f48-t22q5",
      "last_heartbeat": "2024-05-22T20:24:29.753472Z",
      "versions": {
        "deb": "3.3.0.dev",
        "gem": "0.6.0.dev",
        "npm": "0.1.0a5.dev",
        "rpm": "3.26.0.dev",
        "core": "3.54.0.dev",
        "file": "3.54.0.dev",
        "maven": "0.9.0.dev",
        "ostree": "2.4.0.dev",
        "python": "3.12.0.dev",
        "ansible": "0.22.0.dev",
        "certguard": "3.54.0.dev",
        "container": "2.21.0.dev"
      }
    }
  ],
  "online_content_apps": [
    {
      "name": "22@example-pulp-content-66bc7d5fd6-njdz7",
      "last_heartbeat": "2024-05-22T20:24:23.163001Z",
      "versions": {
        "deb": "3.3.0.dev",
        "gem": "0.6.0.dev",
        "npm": "0.1.0a5.dev",
        "rpm": "3.26.0.dev",
        "core": "3.54.0.dev",
        "file": "3.54.0.dev",
        "maven": "0.9.0.dev",
        "ostree": "2.4.0.dev",
        "python": "3.12.0.dev",
        "ansible": "0.22.0.dev",
        "certguard": "3.54.0.dev",
        "container": "2.21.0.dev"
      }
    },
    {
      "name": "24@example-pulp-content-66bc7d5fd6-njdz7",
      "last_heartbeat": "2024-05-22T20:24:23.164853Z",
      "versions": {
        "deb": "3.3.0.dev",
        "gem": "0.6.0.dev",
        "npm": "0.1.0a5.dev",
        "rpm": "3.26.0.dev",
        "core": "3.54.0.dev",
        "file": "3.54.0.dev",
        "maven": "0.9.0.dev",
        "ostree": "2.4.0.dev",
        "python": "3.12.0.dev",
        "ansible": "0.22.0.dev",
        "certguard": "3.54.0.dev",
        "container": "2.21.0.dev"
      }
    }
  ],
  "database_connection": {
    "connected": true
  },
  "redis_connection": {
    "connected": true
  },
  "storage": {
    "total": 132086132736,
    "used": 22736658432,
    "free": 102592557056
  },
  "content_settings": {
    "content_origin": "http://nodeport.local:30000",
    "content_path_prefix": "/pulp/content/"
  },
  "domain_enabled": false
}
```

**Comment by mdellweg** [comment]
What version of pulp-cli-maven is this?
I'm going to make a new release now. But we may want to yank some (all?) old versions.

**Comment by mdellweg** [comment]
I released 0.3.0 (not compatible with pulp-cli 0.25) and 0.3.1 (compatible) and yanked all older releases.
